### PR TITLE
let setDefaultsPTB take care of... setting the audio defaults... 

### PR DESCRIPTION
**Pull request by Remi-Gau**
as it should

### CODE REVIEW

File: src/utils/setDefaultsPTB.m [hunk 1]
```diff
@@ -48,6 +48,9 @@
 
     if isfield(cfg, 'audio') && cfg.audio.do
 
+        cfg.audio.devIdx = [];
+        cfg.audio.playbackMode = 1;
+
```
Comment (CerenB) [inline src/utils/setDefaultsPTB.m:53]: here it would not be `fieldToSet.audio.devIdx = [];` instead of `cfg` ?

**Review by CerenB — APPROVED** [review]

File: src/utils/setDefaultsPTB.m (hunk 1 above)
Comment (Remi-Gau) [reply src/utils/setDefaultsPTB.m:53]: ha yes.

Thanks for spotting that one !! :-)

**Review by CerenB — APPROVED** [review]
then i guess it's ready to be merged and not causing any problems!

**Comment by Remi-Gau** [comment]
> then i guess it's ready to be merged and not causing any problems!

Waiting for the last green light from Travis and then on we go.

**Pull request merged**